src/calculation/DetailedResultsBuilder.ts

```diff
@@ -52,14 +52,35 @@ export function createPopulationValues(
       // create patient level population and stratifier results based on episodes
       episodeResults.forEach(episodeResult => {
         episodeResult.populationResults.forEach(popResult => {
-          createOrSetResult(
-            popResult.populationType,
-            popResult.result,
-            populationResults,
-            popResult.criteriaExpression,
-            popResult.populationId,
-            popResult.criteriaReferenceId
+          const measureObsPop = populationResults.find(
```

Comment:
Not related to this diff, but we actually have a TODO right above this block to collate the observation results at the root, which you are now doing, so we can delete it :)

Reply:
Oh there's also one here: https://github.com/projecttacoma/fqm-execution/blob/13ee8487597076a311a32ee0e818eccf34ffd50f/src/calculation/DetailedResultsBuilder.ts#L299

Reply:
you beat me to it @mgramigna 
![Screen Shot 2022-11-14 at 3 46 42 PM](https://user-images.githubusercontent.com/87094479/201762292-9ccd211b-7b54-468a-930b-72e282ff3f97.png)